.pre-commit-config.yaml

```diff
@@ -5,12 +5,14 @@ default_language_version:
 
 repos:
   - repo: https://github.com/psf/black
-    rev: 22.3.0
+    rev: 23.1.0
     hooks:
       - id: black
         name: (BF) black
-        args: [--preview]
         files: ^backend/benefit
+  - repo: https://github.com/psf/black
+    rev: 22.3.0
```

Comment:
Shouldn't this be black 23.1.0 here also? It would be good to keep black, isort and flake8 at the same version number for all projects and environments in the yjdh monorepo IMHO.

Reply:
Hmm, maybe I tried to upgrade it on tet and kesäseteli side too, but there was some dependency mismatch or something. I will try again.

The new black version formats things a bit differently. If it's updated, then there will be a lot of formatting changes.

Reply:
Formatting changes are expected and ok with black/isort/flake8 updates. It'd be good to have the formatting changes themselves in a separate commit to make it clear that this commit only does formatting changes.